smarsy/bs_helper.py

```diff
@@ -0,0 +1,45 @@
+from bs4 import BeautifulSoup
+
+
+class BSHelper(object):
+    """
+     The help class for BeautifulSoup library
+     """
+    def __init__(self, html):
+        self.html = html
+
+    @property
+    def get_bs_object(self):
```

Comment:
properties doesn't start with `get_` normally. If there is getter - `get_name` so property normally is called just `name`

Reply:
Done